synapse/storage/controllers/persist_events.py

```diff
@@ -716,8 +716,6 @@ async def _persist_event_batch(
                             )
                             if not is_still_joined:
                                 logger.info("Server no longer in room %s", room_id)
-                                latest_event_ids = set()
-                                current_state = {}
```

Comment:
These are both always set higher in this loop and never used after this point in the loop?